Asgardia.Constitution.Proposal.txt

```diff
@@ -155,42 +154,40 @@ ASGARDIAN RESOURCES
 Article 11. Human Resources
 § 1. Asgardia encourages the development of human resources.
 § 2. Asgardia uses its own material, financial, and other resources to ensure free access to distance learning, professional qualifications, and scientific and creative work for Asgardian citizens.
-§ 3. Asgardian citizens are entitled to free time for self-development as well as self-improvement, creative and cultural pursuits for a minimum of 25 percent of the year while employed by a business operating within Asgardian territories. It is not required to spend this time contiguously.
-§ 4. Asgardia works to create jobs and facilitate maximum utilisation of the abilities and qualifications of its citizens.
+§ 3. Asgardian citizens are entitled to free time for self-development, self-improvement, and creative and cultural pursuits for a minimum of 35 percent of days in the Earth year while employed by a business operating within Asgardian territories. It is not required to spend this time contiguously.

```

Comment:
Okay, I'm fine with that. I originally wrote _50%_ ! 
But, then I felt it could be an issue for businesses.